scripts/deployKetherNFT.js

```diff
@@ -50,12 +51,14 @@ async function main() {
   if (account === undefined) {
     throw "Signer account not provided, specify ACCOUNT_PRIVATE_KEY";
   } else if (account.address !== cfg.ketherNFTOwnerAddress) {
-    throw "Did not acquire signer for owner address: " + cfg.ketherNFTOwnerAddress + "; got: " + account.address;
+   // throw "Did not acquire signer for owner address: " + cfg.ketherNFTOwnerAddress + "; got: " + account.address;
```

Comment:
TODO: Resolve this to a better check

Reply:
crap I commented this out for myself but thought I didn't push it.

Reply:
@shazow should I put this back in or add a todo about some check here?

Reply:
Put it back imo